### PR TITLE
sapi: change version to the latest defined by spec

### PR DESCRIPTION
**Pull request by tstruk**
Currently the spec defines ABI version to TSS2_API_VERSION_1_2_1_108.
Update rest of the headers respectively.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -148,7 +148,7 @@ EXTRA_DIST = \
     lib/sapi.pc.in \
     man/man-postlude.troff \
     man/InitDeviceTcti.3.in \
-    man/man3/InitSocketTcti.3 \
```
Comment (martinezjavier) [inline Makefile.am]: I don't think this belong to this commit, right? Instead you should cherry-pick from 1.x branch commit 784695d69c6e ("build: Distribute the source for the InitSocketTcti man page.").

**Comment by martinezjavier** [comment]
@tstruk there seems to be a lot of style cleanups interleaved with the actual changes. Can you please split those in a different commit, otherwise makes it hard to read the commit diff.

File: Makefile.am (hunk 1 above)
Comment (tstruk) [reply Makefile.am]: Yes, that's some leftovers from Philip's PR that somehow got into my branch. Let me rebase. Thanks for catching that @martinezjavier

**Comment by tstruk** [comment]
That garbage issue should be fixed now

**Comment by tstruk** [comment]
@martinezjavier there are some style fixes and ABI version number updates. It's all of the header cleanup activity that we want to get this week to fix the open issue against headers starting from #674. I'll try to separate them for the next headers.

**Comment by martinezjavier** [comment]
@tstruk fair enough. The changes looks good to me, but I still think that at least the commit message should mention the style cleanups. Something like:

```
Currently the spec defines ABI version to TSS2_API_VERSION_1_2_1_108.
Update rest of the headers respectively.

While being there, do some style cleanups and use C style comments instead of C++.
```

**Comment by tstruk** [comment]
Comment updated.

**Pull request merged**